lib/grr.js

```diff
@@ -1552,85 +1571,83 @@ function stepReviewCRs(arg, cb) {
                     }
                 }, function (err, res, body) {
                     if (err) {
-                        next(new VError(err, 'could not retrieve JIRA patch %s',
-                            patch_url));
+                        next(new VError(err,
+                            'could not retrieve gerrit patch %s', patch_url));
                         return;
                     } else if (res.statusCode === 404) {
-                        next(new VError(err, 'no such JIRA patch %s',
+                        next(new VError(err, 'no such gerrit patch %s',
                            patch_url));
                         return;
                     } else if (res.statusCode !== 200) {
                         next(new VError(
-                            'unexpected JIRA response status for patch %s: %s',
+                            'unexpected gerrit response code for patch %s: %s',
                             patch_url, res.statusCode));
                         return;
                     }
-                    try {
-                        var buf = Buffer.from(body, 'base64');
-                        console.log('patch:');
-                        console.log(buf.toString('utf8'));
-                        if (arg.dryRun) {
-                            console.log('Dry run, will not prompt for review.');
-                            return next();
+
+                    var buf = Buffer.from(body, 'base64');
+                    console.log('patch:');
+                    console.log(buf.toString('utf8'));
+                    if (arg.dryRun) {
+                        console.log('Dry run, will not prompt for review.');
+                        return next();
+                    }
+                    var review_desc = extsprintf(
+                        'Do you want to give a \'%s\' +1 for this review?',
+                        arg.reviewArg);
+                    // prompt for user input, and apply CR
+                    var prompt_schema = {
+                        properties: {
+                            answer: {
+                                description: review_desc,
+                                pattern: /[y|n]/,
```

Comment:
I think you need `/^[yn]$/`.  The pipe is wrong in a character class (square brackets) and the anchors are needed else:

```
Do you want to give a 'Code-Review' +1 for this review?: asdf
error:   y or n will do!
Do you want to give a 'Code-Review' +1 for this review?: wowy
skipping this review.
```

Reply:
D'oh, thanks. That works.